metadata/datasets/goa.yaml

```diff
@@ -756,15 +756,15 @@ datasets:
     - NCBITaxon:9031
 
  -
-   id: goa_chicken_rna.gaf
-   label: "goa_chicken_rna gaf file"
-   description: "gaf file for goa_chicken_rna from EBI Gene Ontology Annotation Database"
-   url: http://geneontology.org/gene-associations/goa_chicken_rna.gaf.gz
+   id: goa_chicken_rna_noiea.gaf
+   label: "goa_chicken_rna_noiea gaf file"
+   description: "gaf file for goa_chicken_rna_noiea from EBI Gene Ontology Annotation Database"
+   url: http://geneontology.org/gene-associations/goa_chicken_rna_noiea.gaf.gz
    type: gaf
    dataset: goa_chicken_rna
```

Comment:
Does this also want to be `goa_chicken_rna_noiea`?

Reply:
yes